web3_proxy/src/frontend/mod.rs

```diff
@@ -85,6 +85,7 @@ pub async fn serve(port: u16, proxy_app: Arc<Web3ProxyApp>) -> anyhow::Result<()
             get(users::user_stats_aggregated_get),
         )
         .route("/user/stats/detailed", get(users::user_stats_detailed_get))
+        .route("/user/modify_role", get(users::admin_change_user_roles))
```

Comment:
let's put this at `/admin/modify_role`. Regular users should not have access to do this.